simple-api/README.md

```diff
@@ -1,2 +1,165 @@
+
+> print "Starting the simple-api package"
+Starting the simple-api package
+
+> add_service service_name="kurtosis-simple-api" config=ServiceConfig(image="kurtosistech/kurtosis-simple-api", ports={"http": PortSpec(number=8080, transport_protocol="TCP", application_protocol="http")})
+Service 'kurtosis-simple-api' added with service UUID 'ddf04ab5ba64465d81de661a91aa7226'
+
+> wait recipe=GetHttpRequestRecipe(port_id="http", service_name="kurtosis-simple-api", endpoint="/health", extract="") field="code" assertion="IN" target_value=[200] timeout="30s"
+Wait took 3 tries (2.13371021s in total). Assertion passed with following:
+Request had response code '200' and body "{\"status\":\"healthy\"}"
+
+Starlark code successfully run. No output was returned.
+INFO[2023-02-27T16:21:41-05:00] ========================================================= 
+INFO[2023-02-27T16:21:41-05:00] ||          Created enclave: kurtosis-enclave          || 
+INFO[2023-02-27T16:21:41-05:00] ========================================================= 
+
+```
+
+The logs show how the Kurtosis CLI is retrieving the Kurtosis simple API image that has already been uploaded to DockerHub 
+as `kurtosistech/kurtosis-simple-api` and hereafter running the code in the Starlark script in `kurtosis-package/main.star`.
+The code in the Starlark script first creates the service and afterwards waits until the service health check endpoint returns `200` (i.e. healthy).
+
+4. Once the Kurtosis CLI finishes successfully you can retrieve the logs of the service by running this command:  
+
+```shell
+$ kurtosis service logs kurtosis-enclave kurtosis-simple-api
+```
+
+which will produce an output as this:
+
+```shell
+$ kurtosis service logs kurtosis-enclave kurtosis-simple-api
+Server listening on the port::8080
+```
+
+5. Lastly we can inspect the contents of the enclave more closely by calling the `enclave inspect` command:
+
+```shell
+$ kurtosis enclave inspect kurtosis-enclave
+```
+
+which produces an output similar to this:
+
+```shell
+$ kurtosis enclave inspect kurtosis-enclave
+UUID:                                 4d54d9238d72
+Enclave Name:                         kurtosis-enclave
+Enclave Status:                       RUNNING
+Creation Time:                        Mon, 27 Feb 2023 16:21:36 EST
+API Container Status:                 RUNNING
+API Container Host GRPC Port:         127.0.0.1:50122
+API Container Host GRPC Proxy Port:   127.0.0.1:50123
+
+========================================== User Services ==========================================
+UUID           Name                  Ports                                      Status
+ddf04ab5ba64   kurtosis-simple-api   http: 8080/tcp -> http://127.0.0.1:50137   RUNNING
+```
+
+We can see that the `kurtosis-simple-api` service is running on port 8080 inside the enclave and that it's mapped to port `50137` outside of Docker.
```

Comment:
correction here; its running on 8080 on the host on which the container is running

within the enclave itself you can have multiple things that look like http: 8080/tcp ( the mapping being on the container and nothing to do with the enclave itself)

Reply:
Good point I'll add that